### ISSUE TITLE
channels

### ISSUE THREAD

**Issue by lyg385** [YichenTang97/BiHDM_pytorch#1]
There are 62 channels in the paper, but there are 64 channels in the code. Could you give me a contact information and let's exchange questions.

**Comment by YichenTang97** [comment]
Hi, thanks for the comment. This is not the official implementation for the BiHDM model (I didn't find an official repo so decided to implement and share it on GitHub). 

I used Biosemi Active Two in my studies, and I don't have access to the SEED and MPED datasets the authors tested the model on. (This means the implementation does not guarantee replication of the results reported by the authors. Do expect possible differences from the raw model proposed in the paper). I simply adopted the channel settings in the paper to work on my own datasets and kept these settings in my example code. 

BiHDMClassifier class, however, allows customized channels and stream settings. By modifying "ch_names", "lh_chs", "rh_chs", "lv_chs", and "rv_chs", one can easily adopt this model to various EEG settings.
